react/utils/pages.ts

```diff
@@ -245,6 +245,15 @@ function routeIdFromPath(path: string, routes: Pages) {
   return id
 }
 
+export function warnUserLand(){
+  const runtime = window.__RUNTIME__
+  const shouldWarnUserLand = !runtime.production && runtime.query
```

Comment:
Do something like the `mergePersistingQueries` to avoid rewriting the name of the query